### PR TITLE
Don't delete default network in region quota override example

### PR DESCRIPTION
**Pull request by rileykarson**
With no billing account, this call fails. Test added in https://github.com/GoogleCloudPlatform/magic-modules/pull/4507

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

**Review by c2thorn — APPROVED** [review]

File: mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb [hunk 1]
```diff
@@ -17,4 +16,4 @@ resource "google_service_usage_consumer_quota_override" "override" {
   limit          = "%2Fproject%2Fregion"
   override_value = "10"
```
Comment (rileykarson) [inline mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb:17]:
```suggestion
  override_value = "8"
```

**Comment by rileykarson** [comment]
My first run of this flaked, but it send the same requests to the API as a passing test. I ran it again and it didn't so I feel safe merging. I think that project deletion takes time, and I just got unlucky on my previous run.

**Comment by c2thorn** [comment]
> My first run of this flaked, but it send the same requests to the API as a passing test. I ran it again and it didn't so I feel safe merging. I think that project deletion takes time, and I just got unlucky on my previous run.

Yep, made https://github.com/hashicorp/terraform-provider-google/issues/8608 to address

**Pull request merged**